api/src/main/resources/workbench.yaml

```diff
@@ -184,6 +276,43 @@ definitions:
     description: levels of access to data in the curated data repository
     enum: &DATA_ACCESS_LEVEL [registered, protected]
 
+  WorkspaceListResponse:
+    type: "object"
+    required:
+     - "items"
+    properties:
+      items:
+        type: "array"
+        items:
+           $ref: "#/definitions/Workspace"
+
+  Workspace:
+    type: "object"
+    required:
+     - "name"
+     - "cohorts"
+    properties:
+      id:
+        type: string
+      name:
+        type: string
+      namespace:
+        type: string
+      description:
+        type: string
+      creator:
+        type: string
+      creationTime:
+        type: string
+        format: date-time
+      lastModifiedTime:
+        type: string
+        format: date-time
+      cohorts:
```

Comment:
+1. I don't envision a UI that would show all cohorts in a list of workspaces inline... this resource should just have metadata for workspaces themselves.